src/Microsoft.DotNet.Build.Tasks/PackageFiles/sign.targets

```diff
@@ -34,7 +35,7 @@
   </PropertyGroup>
 
   <Target Name="OpenSourceSign"
-          Condition="'$(DelaySign)' == 'true' and '@(IntermediateAssembly)' != '' and '$(SkipSigning)' != 'true' and '$(UseOpenSourceSign)' == 'true'"
+          Condition="'$(DelaySign)' == 'true' and '@(IntermediateAssembly)' != '' and '$(SkipSigning)' != 'true' and '$(SignType)' == 'oss'"
```

Comment:
I've moved coreclr to the latest buildtools version (193) and it looks like everything builds correctly, so then there is no need to address my previous comment. Once you merge this and we have a new buildtools package, I'll use that from coreclr.